### DISCUSSION TITLE
Allowed Html

### DISCUSSION THREAD

**Discussion by geomodules1** [geodesicsolutions-community/geocore-community#206] [General]
I turned off everything on the Allowed Html form and when placing ad everything shows on the html editor. So the Allowed html form does not work at all

**Comment by blufyremedia** [comment]
Hello,

The backend settings for "Allowed HTML" provide additional means to filter/scrub certain data from user input content. Those settings do not directly influence the TinyMCE HTML editor features/buttons, which **can be modified** in your site's **template set**.
